### DISCUSSION TITLE
Is it possible to limit the number of lambda running concurrently ?

### DISCUSSION THREAD

**Discussion by FinPlorer** [vercel/vercel#5525] [Help] [answered]
Hello everyone, 

Working with nextjs / vercel is amazing ! However when we try to use a Database with a small connection limit, it can become a nightmare. Each lambda create a new database connection which lead to errors like "max connection limit".
One solution is to use a layer for managing a database connection pool.
But an other would be to limit the number of lambda (1000 by default), it seems like it is possible on amazon cloud, so is it also possible to do it with Vercel ?

Thanks

**Comment by mcsdevv** [comment, marked as answer]
Hello @MatthieuCoelho, we recommend either using a serverless ready database (something like Fauna, AWS Aurora) or to handle connections with pooling, DigitalOcean offers this out the box with Postgres.

Further information is available here: https://vercel.com/docs/solutions/databases

**Reply by FinPlorer** [reply]
Thanks @mcsdevv, of course using a serverless ready database is the best way to solve this issue.

So you confirm that there is no way in vercel to limit the number of lambda running concurrently ?

**Reply by mcsdevv** [reply]
That's correct, there is no way to limit this. However, it can be increased on Enterprise plans.
